nilearn/datasets/func.py

```diff
@@ -2371,12 +2402,22 @@ def patch_openneuro_dataset(file_list):
     REPLACEMENTS = {
         "_T1w_brainmask": "_desc-brain_mask",
         "_T1w_preproc": "_desc-preproc_T1w",
-        "_T1w_space-MNI152NLin2009cAsym_brainmask": "_space-MNI152NLin2009cAsym_desc-brain_mask",  # noqa 501
-        "_T1w_space-MNI152NLin2009cAsym_class-": "_space-MNI152NLin2009cAsym_label-",  # noqa 501
-        "_T1w_space-MNI152NLin2009cAsym_preproc": "_space-MNI152NLin2009cAsym_desc-preproc_T1w",  # noqa 501
+        "_T1w_space-MNI152NLin2009cAsym_brainmask": (
+            "_space-MNI152NLin2009cAsym_desc-brain_mask"
+        ),  # noqa 501
+        "_T1w_space-MNI152NLin2009cAsym_class-": (
+            "_space-MNI152NLin2009cAsym_label-"
+        ),  # noqa 501
+        "_T1w_space-MNI152NLin2009cAsym_preproc": (
+            "_space-MNI152NLin2009cAsym_desc-preproc_T1w"
+        ),  # noqa 501
         "_bold_confounds": "_desc-confounds_regressors",
-        "_bold_space-MNI152NLin2009cAsym_brainmask": "_space-MNI152NLin2009cAsym_desc-brain_mask",  # noqa 501
-        "_bold_space-MNI152NLin2009cAsym_preproc": "_space-MNI152NLin2009cAsym_desc-preproc_bold",  # noqa 501
+        "_bold_space-MNI152NLin2009cAsym_brainmask": (
+            "_space-MNI152NLin2009cAsym_desc-brain_mask"
+        ),  # noqa 501
+        "_bold_space-MNI152NLin2009cAsym_preproc": (
+            "_space-MNI152NLin2009cAsym_desc-preproc_bold"
+        ),  # noqa 501
```

Comment:
we should now be able to remove all the ignore E501 errors from all of these right?

Reply:
yup